crypto/keys/keybase.go

```diff
@@ -426,6 +426,11 @@ func (kb dbKeybase) Update(name, oldpass string, getNewpass func() (string, erro
 	}
 }
 
+// CloseDB releases the lock and closes the storage backend.
+func (kb dbKeybase) CloseDB() {
```

Comment:
Should we call this (e.g. in the code) in places other than the testcase you added? Seems like maybe so...

Reply:
Yeah maybe